README.md

```diff
@@ -6,92 +6,25 @@
 
 Software Development Kit for interacting with Polygon Nightfall.
 
-You can clone the repo and [play with the example scripts](#play-with-the-sdk-repository), or you can [install it as a dependency](#install-sdk-from-npm) from NPM.
+You can clone the repo and [play with the example scripts](#play-with-the-sdk-repository), or you can [install it as a dependency](https://wiki.polygon.technology/docs/nightfall/tools/user-sdk-getting-started) from NPM.
 
 More about [Polygon Nightfall](https://docs.polygon.technology/docs/nightfall/introduction/overview).
 
 ### What is the SDK for?
 
-- Deposit ERC20 tokens from Ethereum L1 to Polygon Nightfall L2
-- Transfer ERC20 token commitments on Polygon Nightfall L2
-- Withdraw ERC20 token commitments from Polygon Nightfall L2 to Ethereum L1
-- Check ERC20 balances on Polygon Nightfall L2
-- Safely export/import ERC20 transaction commitments from and to Polygon Nightfall L2
+- Deposit ERC20,ERC721 and ERC1155 tokens from Ethereum L1 to Polygon Nightfall L2
+- Transfer ERC20,ERC721 and ERC1155 token commitments on Polygon Nightfall L2
+- Withdraw ERC20,ERC721 and ERC1155 token commitments from Polygon Nightfall L2 to Ethereum L1
+- Check ERC20,ERC721 and ERC1155 balances on Polygon Nightfall L2
+- Safely export/import ERC20,ERC721 and ERC1155 transaction commitments from and to Polygon Nightfall L2
 
 To learn more about transactions, commitments and other core Nightfall features go to the [Protocol Docs](https://docs.polygon.technology/docs/category/nightfall-protocol/).
 
 ## What is a Nightfall Client?
 
-The Nightfall Client is a key part of the architecture. It generates zero-knowledge proofs, stores zero-knowledge commitments and interacts with the contracts. The SDK uses Nightfall Client to interact with the Nightfall Protocol.
+The Nigthfall Client is a key component that the SDK can't work without
```

Comment:
I'd rather say e.g. "To use the SDK you need to spin a Nightfall Client" - more straight fw than using that double "negative" cant + without... Hopefully you see what I mean!! :pray:

Reply:
yeah makes sense, i was trying to be formal lol